model/expansions.go

```diff
@@ -0,0 +1,25 @@
+package model
+
+import (
+	"github.com/evergreen-ci/evergreen/model/patch"
+	"github.com/evergreen-ci/evergreen/util"
+	"github.com/pkg/errors"
+)
+
+// UpdateExpansions first updates expansions with project variables.
+func UpdateExpansions(expansions *util.Expansions, projectId string, params []patch.Parameter) error {
```

Comment:
I would move this into host_create. I don't think it's generally useful and it was private before

Reply:
👍